tests/apichecks/testdata/exceptions/shortname_pluralization.txt

```diff
@@ -1 +1,2 @@
+[shortname_plural] crd=apiquotaadjustersettings.cloudquota.cnrm.cloud.google.com version=v1alpha1: plural shortName "gcpapiquotaadjustersettingss" appears to have incorrect pluralization of "gcpapiquotaadjustersettings"
```

Comment:
Please fix this.

Reply:
i dont know what to have for the plural. Do we change the singular to setting ? Should we change all the go types ? And regenerate from scratch ? 
Or just the CRD names.

Reply:
done.